### PR TITLE
Added retries for httpd.conf

### PR DESCRIPTION
**Pull request by Sharvaribarve**
Added 3 retries for httpd.conf, since otherwise it gives an intermittent error of the browser not rendering the text

### CODE REVIEW

File: jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/steps/operation/WebServerControlRunSteps.java [hunk 1]
```diff
@@ -46,6 +47,26 @@ public void clickWebServerControlLink(final String linkLabel, final String webSe
                 + webServerName + "']/following-sibling::td//button[text()='" + linkLabel + "']"));
     }
 
+    @When("^I click the httpd.conf link of web server \"(.*)\" under group \"(.*)\" in the operations tab and I see httpd.conf$")
+    public void clickWebServerHttpdConfLink(final String webServerName, final String groupName) {
+        origWindowHandle = jwalaUi.getWebDriver().getWindowHandle();
+        int count = 0;
+        while(count < 3) {
```
Comment (jmahmoodcc) [inline jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/steps/operation/WebServerControlRunSteps.java:54]: We should make the max number of retries a configurable property with a default value of 3

File: jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/steps/operation/WebServerControlRunSteps.java [hunk 2]
```diff
@@ -60,13 +81,14 @@ public void verifyIfDeleteWebServerIsSuccessful(final String webServerName) {
     }
 
     @Then("^I see the httpd.conf$")
-    public void verifyProperConfFile() {
+    public boolean verifyProperConfFile() throws TimeoutException{
```
Comment (jmahmoodcc) [inline jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/steps/operation/WebServerControlRunSteps.java:84]: If this method is no longer used in any other feature we should make it private and remove the @Then annotation

**Review by jmahmoodcc — COMMENTED** [review]
I'm not sure how I feel about this fix in general - to me it looks like we're masking an intermittent issue. What's the consensus from the rest of the team?

**Comment by jeddcuison** [comment]
Not to fan the flame or anything, I agree with Jeff that this might be an issue and we should probably fix it. Does it happen in Jwala ? Like when you click httpd.conf manually it does not show up sometimes ?

**Pull request closed without merging**